plugins/shell/src/process/mod.rs

```diff
@@ -187,14 +190,19 @@ impl Command {
         K: AsRef<OsStr>,
         V: AsRef<OsStr>,
     {
-        self.0.envs(envs);
+        self.cmd.envs(envs);
         self
     }
 
     /// Sets the working directory for the child process.
     #[must_use]
     pub fn current_dir<P: AsRef<Path>>(mut self, current_dir: P) -> Self {
-        self.0.current_dir(current_dir);
+        self.cmd.current_dir(current_dir);
+        self
+    }
+    
+    pub fn set_raw_out(mut self, raw_out: bool) -> Self {
```

Comment:
Missing documentation and you will also need to update the documentation of `CommandEvent` enum

Reply:
Done